platforms/jvm/jacoco/src/testFixtures/groovy/org/gradle/testing/jacoco/plugins/fixtures/JacocoCoverage.groovy

```diff
@@ -24,7 +24,7 @@ final class JacocoCoverage {
 
     private JacocoCoverage() {}
 
-    private static final String[] ALL = [JacocoPlugin.DEFAULT_JACOCO_VERSION, '0.7.1.201405082137', '0.7.6.201602180812', '0.8.3'].asImmutable()
+    private static final String[] ALL = [JacocoPlugin.DEFAULT_JACOCO_VERSION, '0.7.6.201602180812'].asImmutable()
```

Comment:
Shouldn't this list more versions? We're only testing two with this?

Reply:
Is it necessary? I would really only want coverage of things we know are used or problematic, and 0.7 is so old (the listed version is 2016!) that I honestly considered removing it as well, but figured there's probably something out there that still uses it.

Reply:
I guess this is OK (the 0.7.x version represents the lowest supported version in a way).  I was thinking of other cases where we have a variety of versions because there are new features that are only supported in certain versions and up.  We don't do that with JaCoCo